code/src/cljs/sixsq/nuvla/ui/plugins/tab_new.cljs

```diff
@@ -0,0 +1,88 @@
+(ns sixsq.nuvla.ui.plugins.tab-new
```

Comment:
`tab-new` `tab-old` aren't good names for later on.
Is it possible to merge them and activate navigation by query-params optionally.

Reply:
`tab-new` should have been the name during development: I wanted to remove it after changing all usages to this new tab plugin. After I couldn't use it in deployment-sets I forgot to give it a better name.

Merging it is too much work I think: Same subs and events but very different logic. Using separate namespaces seems cleaner to me

Two suggestions:
1. How about naming the new tab plugin `nav-tab`?
2. I create a new ui issue for refactor deployment sets when it's finished, sot that it uses the new tab plugin and to remove the old plugin.